201-eventhubs-create-namespace-and-enable-archive/azuredeploy.JSON

```diff
@@ -0,0 +1,152 @@
+{  
+   "$schema":"http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
```

Comment:
Is it possible to use a non-preview schema instead? e.g. https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#